bundles/org.openhab.binding.kaleidescape/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -376,6 +380,17 @@
 		<state readOnly="true" pattern="%d %unit%"/>
 	</channel-type>
 
+	<channel-type id="endtime">
+		<item-type>DateTime</item-type>
+		<label>Title End Time</label>
+		<description>The date/time when the currently playing movie will end</description>
+		<category>Time</category>
+		<tags>
+			<tag>Timestamp</tag>
+		</tags>
```

Comment:
You can't set only a property, you have to combine it with a Point property.

Reply:
Thanks. There are many examples of this without Point in the shelly binding.

Reply:
It was not defined as expected in the Shelly binding !

Rather use a more specific value. I hesitate between "Status" and "Measurement".